web/src/components/TtsPlayer.tsx

```diff
@@ -0,0 +1,177 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled, { useTheme } from 'styled-components'
+
+import { CloseIcon, PauseIcon, PlaybackIcon, PlayIcon } from '../assets'
+import dimensions from '../constants/dimensions'
+import TtsHelpModal from './TtsHelpModal'
+import Button from './base/Button'
+import Icon from './base/Icon'
+
+const StyledTtsPlayer = styled.dialog<{ $isPlaying: boolean }>`
+  background-color: ${props => props.theme.colors.ttsPlayerBackground};
+  border-radius: 28px;
+  width: 388px;
+  display: flex;
+  flex-direction: ${props => (props.$isPlaying ? 'column' : 'row')};
+  justify-content: center;
+  align-items: center;
+  padding: 6px;
+  position: sticky;
+  bottom: 5px;
+  min-height: 93px;
+  gap: ${props => (props.$isPlaying ? '5px;' : '10px')};
+  border-color: transparent;
```

Comment:
I don't think this line does anything

Reply:
without it:
![image](https://github.com/user-attachments/assets/03e9f9eb-b903-453b-836c-24548c8f2bca)